content/env-variables.md

```diff
@@ -220,20 +220,20 @@ pub struct UpdateProgramConfig<'info> {
 }
 ```
 
-Before instruction logic even executes, a check will be performed to make sure the instruction's signer matches the hard-coded `ADMIN_PUBKEY`. Notice that the example above doesn't show the instruction that initializes the config account, but it should have similar constraints to ensure that an attacker can't initialize the account with unexpected values.
+Before instruction logic executes, a check will be performed to ensure the instruction's signer matches the hard-coded `ADMIN_PUBKEY`. Notice that the example above doesn't show the instructions for initializing the config account, but it should have similar constraints to ensure that an attacker can't initialize the account with unexpected values.
 
 While this approach works, it also means keeping track of an admin wallet on top of keeping track of a program's upgrade authority. With a few more lines of code, you could simply restrict an instruction to only be callable by the upgrade authority. The only tricky part is getting a program's upgrade authority to compare against.
 
-### Constrain config updates to the program's upgrade authority
+### Constraint config updates to the program's upgrade authority
```

Comment:
This should still be Constrain - it's meant to be a verb

Reply:
Sorry, my bad :(